wordstats/language_data/t2l/code/translations.py

```diff
@@ -0,0 +1,123 @@
+from query import query_site
+from pathlib import Path
+import time
+import math
+import random
+
+"""
+
+Class that calls a translation API to translate words and stores the
+translations as a txt file
+
+"""
+
+# First word from which to start translating
+BEGIN = 0
+
+# How many words to translate before storing them (the last one is not included)
+BATCH = 700
+
+FROM_CODE = "nl"
+DEST_CODE = "en"
+
+# Path of the txt file with the words
+PATH_INPUT_WORDS = str(Path(__file__).parent.parent) + '/data/words/' + FROM_CODE + '.txt'
+
+# Path of the txt file with the translations
+PATH_OUTPUT_TRANSLATIONS = str(Path(__file__).parent.parent) + "/data/translations/" + FROM_CODE + "-" + \
```

Comment:
It is very curious, because in my computer works perfectly like that. Maybe I had to install something but I don't remember, so two options:

- Search for what I installed and updated in the README
- Change the Path thing to the way you said.

Tell me what you prefer